test/automated/api/run.sh

```diff
@@ -6,14 +6,13 @@ TEMP_DB=$(mktemp)
 npm install --quiet --no-progress
 
 # Download a specific version of ffmpeg
-if [ ! -d "ffmpeg" ]; then
-	mkdir ffmpeg
-	pushd ffmpeg >/dev/null || exit
-	curl -sL https://github.com/vot/ffbinaries-prebuilt/releases/download/v4.2.1/ffmpeg-4.2.1-linux-64.zip --output ffmpeg.zip >/dev/null
-	unzip -o ffmpeg.zip >/dev/null
-	PATH=$PATH:$(pwd)
-	popd >/dev/null || exit
-fi
+FFMPEG_PATH=$(mktemp -d)
```

Comment:
Does this mean it's now downloading a new copy of ffmpeg to a new place every time the tests run? That's a bit excessive. I often run the tests many times and I'd hate to waste all that time and space downloading the same copy of ffmpeg over and over again.

Reply:
I moved the ffmpeg to a sub-directory. Now, it is only downloaded once.